ci/docker/runtime_functions.sh

```diff
@@ -385,6 +386,28 @@ unittest_ubuntu_python2_gpu() {
     nosetests-2.7 --verbose tests/python/gpu
 }
 
+tutorialtest_ubuntu_python3_gpu() {
+    set -ex
+    cd /work/mxnet/docs
+    export MXNET_DOCS_BUILD_MXNET=0
+    make html
+    export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
+    export PYTHONPATH=/work/mxnet/python/
+    export MXNET_TUTORIAL_TEST_KERNEL=python3
+    cd /work/mxnet/tests/tutorials && nosetests-3.4 test_tutorials.py --nologcapture
+}
+
+tutorialtest_ubuntu_python2_gpu() {
+    set -ex
+    cd /work/mxnet/docs
+    export MXNET_DOCS_BUILD_MXNET=0
+    make html
+    export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
+    export PYTHONPATH=/work/mxnet/python/
+    export MXNET_TUTORIAL_TEST_KERNEL=python2
+    cd /work/mxnet/tests/tutorials && nosetests-3.4 test_tutorials.py --nologcapture
```

Comment:
-> nosetests-2.7

why no log capture?

Reply:
nosetest is trying to be smart but in this case since we are spawning a new process we get a whole load of unwanted extra logging:
- the raw text of the notebook
- the debug print out of jupyter (e.g the heartbeat signals of the kernels)
I am already printing the errors and warning (which are captured even with --nologcapture)